package.json

```diff
@@ -61,9 +61,10 @@
     "url": "https://github.com/MarkBind/markbind.git"
   },
   "scripts": {
+    "csslint": "./node_modules/.bin/stylelint '**/*.css' '!**/*.min.css'",
```

Comment:
This didn't work, I modified `asset/css/markbind.css` to violate some rules (e.g. `0` -> `0px`, `#abcdef` -> `#ABCDEF`), but stylelint didn't complain about anything.

Reply:
Hmm that's odd, it seems to be working for me:

<img width="704" alt="Screenshot 2019-04-12 at 10 40 16 AM" src="https://user-images.githubusercontent.com/17447681/56008634-72987f00-5d0f-11e9-8a53-2f42af91cab8.png">

I'll try to see if I can get hold of a windows machine to replicate this.

Reply:
Tried it on another Windows computer, still doesn't work:

![csslint](https://user-images.githubusercontent.com/3168908/56011704-69adaa80-5d1b-11e9-8be1-bc5cd97e0fee.png)

Reply:
Seems to be related to https://github.com/stylelint/stylelint/issues/3624. I've replaced the single quotes with double quotes - do you mind giving this another try to see if it works @yamgent? Thanks!

Reply:
Works now.